src/routes/viewRoutes.js

```diff
@@ -0,0 +1,13 @@
+//get the functions to show the index.html and notes.html
+const { Router } = require("express");
+
+const { renderHome, renderNotes } = require("../controllers/view");
+
+const newRouter = Router();
+
+//get the routers
+newRouter.get("/notes", renderNotes);
```

Comment:
You are missing the `*` route which will prevent the application from breaking if any path other than `/` and `/notes` is entered.